src/popper/scm.py

```diff
@@ -198,3 +198,11 @@ def get_branch(repo):
         return branch
 
     return get_sha(repo)
+
+
+def get_tag(repo=None):
+    """Obtains the git tag fot the currently checked our version on the given repository object
+        if possible, otherwise returns ""
+    """
+    if repo:
+        return repo.describe('--tags', '--dirty','--broken', '--long')
```

Comment:
sorry for not explaining this better in the issue description. The idea for having a `GIT_TAG` variable is that it would be defined only if the current commit is tagged, so this could work:

```
git tag --points-at $(git rev-parse HEAD)
```

(we'd need to find the way of doing the above with GitPython).

The other issue to keep in mind is that in some scenarios (e.g. in Travis), tags are not fetched when doing a checkout, and instead they are obtained via github and defined like `TRAVIS_TAG`. So if `git tag --points-at` doesn't point return anything, we can check for `CIRCLE_TAG`, `TRAVIS_TAG`, etc. as it is done currently for `git_branch()` so that we can obtain the tag from the CI env variables.